### ISSUE TITLE
how can I turn on the validation mode on train.py

### ISSUE THREAD

**Issue by enginbozaba** [junyanz/pytorch-CycleGAN-and-pix2pix#961]
How can I use both trainA-trainB and valA-valB datasets on train.py? ie how can I turn on the validation mode on train.py?

**Comment by junyanz** [comment]
The program doesn't support this. Maybe you can copy and paste `train` and `val` to a new directory called `trainval` and use the flag `--phase trainval`.

**Comment by hyecheol123** [comment]
So is there any reason why we divide the entire dataset to train/test/val?
Is there any part of code that the program utilizing the validation part of dataset?

**Comment by junyanz** [comment]
There is some confusion about names. Currently, val and test are the same. You can use it as test set.
